File: Frontend/src/interfaces/v12/StockRecieve/StockRecieveDetails/StockRecieveItemRemarkDetailsDto.ts
import { guid } from "types/guid";

export interface StockRecieveItemRemarkDetailsDto {
  id: guid;
  StockRecieveItemId: guid;
  productRemarkId: guid;
  remark: string;
  createdAt?: string;
  createdById?: guid;
  changedAt?: string;
  changedById?: guid;
}

File: Wms.Api/Dto/StockRecieve/StockRecieveDetails/StockRecieveItemRemarkDetailsDto.cs
namespace Wms.Api.Dto.StockRecieve.StockRecieveDetails
{
    public class StockRecieveItemRemarkDetailsDto
    {
        public Guid Id { get; set; }
        public Guid StockRecieveItemId { get; set; }
        public Guid ProductRemarkId { get; set; }
        public string Remark { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public Guid CreatedById { get; set; }
        public DateTime? ChangedAt { get; set; }
        public Guid? ChangedById { get; set; }
    }
}

